dewy/documents/models.py

```diff
@@ -5,9 +5,11 @@
 
 
 class CreateRequest(BaseModel):
-    """The name of the collection the document should be added to."""
+    """The name of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
+    collection: Optional[str] = None
 
-    collection_id: int
+    """The id of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
```

Comment:
Making it easier when using the API clients - it can be a PITA to fetch the collection ID's.